api/management/api.go

```diff
@@ -207,6 +207,7 @@ type RealmCustomConfiguration struct {
 	OnboardingClientID                  *string   `json:"onboarding_client_id"`
 	SelfRegisterGroupNames              *[]string `json:"self_register_group_names"`
 	BarcodeType                         *string   `json:"barcode_type"`
+	BackURL                             *string   `json:"back_url,omitempty"`
```

Comment:
Might be better to name it allowed_back_url to make it more comprehensive of the exact role of it.

Reply:
Done